### ISSUE TITLE
java.io.IOException: Failed to determine protocol version

### ISSUE THREAD

**Issue by crystalztl** [flutter/flutter-intellij#6094]
Failed to connect to the VM observatory service at: [ws://127.0.0.1:65457/RfjYGuLtVVI=/ws]()
java.io.IOException: Failed to determine protocol version

## Steps to Reproduce

Run flutter app in Physical Android device, Pixel 4a.

## Version info

[✓] Flutter (Channel stable, 2.10.3, on macOS 12.3.1 21E258 darwin-x64, locale en-SG)
    • Flutter version 2.10.3 at /Users/xxx/dev/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 7e9793dee1 (6 weeks ago), 2022-03-02 11:23:12 -0600
    • Engine revision bd539267b4
    • Dart version 2.16.1
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/xxx/Library/Android/Sdk
    • Platform android-32, build-tools 31.0.0
    • ANDROID_HOME = /Users/xxx/Library/Android/Sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7590822)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7590822)

[✓] IntelliJ IDEA Ultimate Edition (version 2021.3.3)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 66.0.3
    • Dart plugin version 213.7306

[✓] VS Code (version 1.66.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.38.0

[✓] Connected device (2 available)
    • Pixel 4 (mobile) • 99211FFAZ00754 • android-arm64  • Android 12 (API 32)
    • Chrome (web)     • chrome         • web-javascript • Google Chrome 100.0.4896.75

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

**Comment by crystalztl** [comment]
Flutter run has no issue

**Comment by helin24** [comment]
@crystalztl What version of IntelliJ/Android Studio were you using when this happened?

And did you mean that `flutter run` works on the command line?

**Comment by crystalztl** [comment]
@helin24  IntelliJ IDEA Ultimate Edition (version 2021.3.3). Yes. flutter run works on the command line

**Comment by helin24** [comment]
I'd like to understand if this is a network issue or if it's specific to IntelliJ.

When you use `flutter run`, it probably ends with this:
```
The Flutter DevTools debugger and profiler on iPhone 13 is available at: http://127.0.0.1:9100?uri=http://127.0.0.1:57876/ItWMaFdP2vc=/
```

Are you able to click on the DevTools link and debug your app there? (There is a Debugger tab and you can set breakpoints in files there)
